api/models/productDao.js

```diff
@@ -0,0 +1,19 @@
+const dataSource = require("./dataSource");
+
+const productList = async (categoryId, size, color) => {
+  return await dataSource.query(
+    `SELECT
+            p.name as product_name,
+            p.price as product_price,
+            pi.image_url as product_image,
+            l.id as product_likes
+        FROM products as p
+        LEFT JOIN likes as l ON p.id = l.product_id
+        LEFT JOIN product_images as pi ON p.id = pi.product_id
+        `
+  );
+};
```

Comment:
```suggestion
const productList = async (categoryId, size, color) => {
  return dataSource.query(
    `SELECT
            p.name as product_name,
            p.price as product_price,
            pi.image_url as product_image,
            l.id as product_likes
        FROM products as p
        LEFT JOIN likes as l ON p.id = l.product_id
        LEFT JOIN product_images as pi ON p.id = pi.product_id
        `
  );
};
```
1. await가 필요없는 코드입니다 :)
2. likes의 의도는, 사람들이 좋아요를 누른 총 개수를 표기하기 위함인가요, 아니면 로그인한 유저가 좋아요를 누른 여부를 판단하기 위함인가요?
3. likes는 데이터가 있을 수도, 없을 수도 있기 때문에 LEFT JOIN을 활용하는 것이 용이하나, product_images를 LEFT JOIN한 이유가 있나요? 이미지도 없는 경우가 있을 수 있기 때문인가요?? (프로젝트 기획을 정확히 알지 못해 여쭤봅니다!)
4. 금일 진행되었던 쿼리빌더 세션의 예시 코드를 참고하여 dynamic filtering 기능을 마저 구현해보셨으면 좋겠습니다!

Reply:
likes는 좋아요 기능을 추가하려다가 빼기로 해서 뺄 예정입니다!

그리고 추가로 필터 기능도 추가해서 PR 올리겠습니다~